style.css

```diff
@@ -437,4 +437,382 @@ input[type=email] {
   .fontawesome {
     display: none;
   }
+
+  .uber-navig {
+    display: flex;
+    flex-direction: column;
+    width: 345px;
+    height: 596px;
+    padding: 16px;
+    margin: 25px 24px;
+    border-radius: 16px;
+    border: 1px solid #dfe1e6;
+    box-sizing: border-box;
+    background-color: #fff;
+    justify-content: center;
+  }
+}
+
+@media screen and (min-width: 1000px) {
+  body {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    width: 1600px;
+  }
+
+  .header {
+    height: 280px;
+    background-color: #fff;
+    padding: 25px 155px;
+  }
+
+  #nav-bar {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    margin: 0 auto;
+    padding-top: 20px;
+    align-items: center;
+  }
```

Comment:
- [ ] id is used for single elements that appear on the page for only once. Kindly, let us move our ids to classes.